### PR TITLE
feat(@test): Import seed for existing user

### PR DESCRIPTION
**Pull request by OlufemiAdeOlusile**
feat(@test): Add a display name negative tests
feat(@test): Fix chat flow scenario
feat(@test): Clean up Names.py

fixes https://github.com/status-im/status-desktop/issues/6432
fixes https://github.com/status-im/status-desktop/issues/6435
fixes https://github.com/status-im/status-desktop/issues/6520
fixes https://github.com/status-im/status-desktop/issues/6539

### CODE REVIEW

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py [hunk 1]
```diff
@@ -3,151 +3,84 @@
...
-mainWindow_inputRectangle_Rectangle_2 = {"container": statusDesktop_mainWindow, "id": "inputRectangle", "type": "Rectangle", "unnamed": 1, "visible": True}
-mainWindow_You_will_not_be_able_to_recover_this_password_if_it_is_lost_StatusBaseText = {"container": statusDesktop_mainWindow, "text": "You will not be able to recover this password if it is lost.", "type": "StatusBaseText", "unnamed": 1, "visible": True}
-mainWindow_Ok_got_it_StatusBaseText = {"container": statusDesktop_mainWindow, "text": "Ok, got it", "type": "StatusBaseText", "unnamed": 1, "visible": True}
-mainWindow_background_Rectangle = {"container": statusDesktop_mainWindow, "id": "background", "type": "Rectangle", "unnamed": 1, "visible": True}
-o_Rectangle = {"container": statusDesktop_mainWindow_overlay, "occurrence": 2, "type": "Rectangle", "unnamed": 1, "visible": True}
+username_must_be_at_least_5_characters_error_message = {"container": statusDesktop_mainWindow, "text": "Username must be at least 5 characters", "type": "StatusBaseText", "unnamed": 1, "visible": True}
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/scripts/names.py:27]: Same that said in other occasions.. `text` property should not be used. Instead, we should introduce the expected error message as a test data and in the verification stage, compere the given expected data with the object.text property

File: test/ui-test/src/screens/StatusWelcomeScreen.py [hunk 2]
```diff
@@ -55,12 +55,19 @@ def __init__(self):
 
     def agree_terms_conditions_and_generate_new_key(self):
         self._agree_terms_and_conditions()
+        click_obj_by_name(SignUpComponents.NEW_TO_STATUS.value)
         click_obj_by_name(SignUpComponents.GENERATE_NEW_KEYS.value)
         
     def agree_terms_conditions_and_navigate_to_import_seed_phrase(self):
         self._agree_terms_and_conditions()
+        click_obj_by_name(SignUpComponents.NEW_TO_STATUS.value)
         click_obj_by_name(SeedPhraseComponents.IMPORT_A_SEED_TEXT.value)
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusWelcomeScreen.py:64]: Are we using a `text` object to click into it like a button??

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py [hunk 3]
```diff
@@ -3,151 +3,84 @@
...
-mainWindow_submitBtn_StatusButton = {"container": statusDesktop_mainWindow, "id": "submitBtn", "type": "StatusButton", "unnamed": 1, "visible": True}
-appearance_SettingsMenu_StatusNavigationListItem = {"container": mainWindow_ScrollView, "objectName": "Appearance-SettingsMenu", "type": "StatusNavigationListItem", "visible": True}
-advanced_SettingsMenu_StatusNavigationListItem = {"container": mainWindow_ScrollView, "objectName": "Advanced-SettingsMenu", "type": "StatusNavigationListItem", "visible": True}
-walletSettingsLineButton = {"container": mainWindow_ScrollView_2, "objectName": "WalletSettingsLineButton", "type": "StatusSettingsLineButton", "visible": True}
 navBarListView_Settings_navbar_StatusNavBarTabButton = {"checkable": True, "container": mainWindow_navBarListView_ListView, "objectName": "Settings-navbar", "type": "StatusNavBarTabButton", "visible": True}
 settings_navbar_settings_icon_StatusIcon = {"container": navBarListView_Settings_navbar_StatusNavBarTabButton, "objectName": "settings-icon", "source": "qrc:/StatusQ/src/assets/img/icons/settings.svg", "type": "StatusIcon", "visible": True}
+
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/scripts/names.py:85]: Leftover!

File: test/ui-test/testSuites/suite_status/shared/steps/steps.py [hunk 4]
```diff
@@ -14,13 +14,20 @@
...
 @When("the user restarts the app")
 def step(context):
     waitFor(lambda: currentApplicationContext().detach(), 500)
     time.sleep(5)
     startApplication("nim_status_client")
+    
+
+@When("the app data folder is erased")
+def step(context):
+    erase_directory("../../../../../Status/data")
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/steps/steps.py:30]: We should avoid relative paths like this. Maybe we should create an environment variable with the root path and then build the paths from there.

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature [hunk 5]
```diff
@@ -14,25 +14,27 @@ Feature: Status Desktop Sign Up
...
   Scenario: User signs up and signs in with password
     Given A first time user lands on the status desktop and generates new key
     When user signs up with username tester123 and password TesTEr16843/!@00
     Then the user lands on the signed in app
 
-
+  @journey-test
   Scenario Outline: User cannot sign up with wrong username format
     Given A first time user lands on the status desktop and generates new key
     When user inputs the following <username> with ui-component mainWindow_edit_TextEdit
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:26]: This statement should be avoided and encapsulate the needed steps in a specific screen definition. Same for the next statements.

File: test/ui-test/testSuites/suite_status/shared/steps/steps.py [hunk 6]
```diff
@@ -14,13 +14,20 @@
...
 
+
 @When("the user restarts the app")
 def step(context):
     waitFor(lambda: currentApplicationContext().detach(), 500)
     time.sleep(5)
     startApplication("nim_status_client")
+    
+
+@When("the app data folder is erased")
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/steps/steps.py:28]: I cannot see were is it used. Maybe you could add it into the hook??

**Review by noeliaSD — CHANGES_REQUESTED** [review]

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature (hunk 5 above)
Comment (noeliaSD) [reply test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:26]: Yeah, but I mean in this particular case I see that it could be better to encapsulate the functionality into a specific screen object. WDYT?
